charts/helm-controller-operator/templates/sourceController/service.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "helm-controller-operator.fullname" . }}-metrics-service-source
+  labels:
+    {{- include "helm-controller-operator.labels" . | nindent 4 }}
+spec:
+  ports:
+    {{- if .Values.kubeRBACProxy.enabled }}
+    - name: https-metrics
+      port: 8443
+      targetPort: https-metrics
+      protocol: TCP
+    {{- else }}
+    - name: http-metrics
+      port: 8080
+      targetPort: http-metrics
+      protocol: TCP
+    {{- end }}
+  selector:
+    {{- include "helm-controller-operator.selectorLabels" . | nindent 4 }}
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "helm-controller-operator.fullname" . }}-source
+  labels:
+    {{- include "helm-controller-operator.labels" . | nindent 4 }}
+spec:
+  clusterIP: {{ (index .Values.service.clusterIps 0) }}
+  clusterIPs:
+  {{- range .Values.service.clusterIps }}
+    - {{ . }}
+  {{- end }}
+  internalTrafficPolicy: Cluster
+  ipFamilies:
+  - IPv4
+  ipFamilyPolicy: SingleStack
```

Comment:
All of this is not needed : 
* clusterIP are automatically filled
* internalTrafficPolicy / ipFamilies / ipFamilyPolicy are at their default values. 

You should compare what helm template and the kustomize produces, I'm sure theses were not there.

```suggestion
```